README.md

```diff
@@ -30,6 +30,40 @@ There are plenty of small, self-contained examples [in the documentation][exampl
 
 The easiest way to get started using ct is to download one of the binaries from the [releases page on GitHub][releases].
 
+One can use the following script to download and verify the signature of Config Transpiler:
+
+```bash
+# Specify Config Transpiler version
+CT_VER=v0.6.1
+
+# Specify Architecture
+# ARCH=aarch64 # ARM's 64-bit architecture
+ARCH=x86_64
+
+# Specify OS
+# OS=apple-darwin # MacOS
+# OS=pc-windows-gnu.exe # Windows
+OS=unknown-linux-gnu # Linux
+
+# Specify download URL
+DOWNLOAD_URL=https://github.com/coreos/container-linux-config-transpiler/releases/download
+
+# Remove previous downloads
+rm -f /tmp/ct-${CT_VER}-${ARCH}-${OS} /tmp/ct-${CT_VER}-${ARCH}-${OS}.asc /tmp/coreos-app-signing-pubkey.gpg
+
+# Download Config Transpiler binary
+curl -L ${DOWNLOAD_URL}/${CT_VER}/ct-${CT_VER}-${ARCH}-${OS} -o /tmp/ct-${CT_VER}-${ARCH}-${OS}
+chmod u+x /tmp/ct-${CT_VER}-${ARCH}-${OS}
```

Comment:
Doesn't seem needed?

Reply:
Presumably a user will want to execute ct after they've verified that their copy is signed properly. Are you saying this step is implied and thus we should omit it?

Reply:
Seems fine, though not quite related to doing the gpg verify.

Reply:
To be able to execute `ct` you need to `chmod u+x` it.
The proper rights are missing on the downloaded file:

```bash
$ ls -l /tmp/ct-v0.6.1-x86_64-unknown-linux-gnu                                                 
-rw-r--r-- 1 czerasz czerasz 6158287 Feb 14 13:05 /tmp/ct-v0.6.1-x86_64-unknown-linux-gnu
```